augly/video/augmenters/ffmpeg/base_augmenter.py

```diff
@@ -63,3 +64,41 @@ def add_augmenter(
             applied and a dictionary with any output arguments as necessary
         """
         raise NotImplementedError("Implement add_augmenter method")
+
+
+class BaseVidgearFFMPEGAugmenter(ABC):
+    def add_augmenter(
+        self, video_path: str, output_path: Optional[str] = None, **kwargs
+    ) -> None:
+        """
+        Applies the specific augmentation to the video
+
+        @param video_path: the path to the video to be augmented
+
+        @param output_path: the path in which the resulting video will be stored.
+        If not passed in, the original video file will be overwritten
+        """
+        video_path, output_path = validate_input_and_output_paths(
+            video_path, output_path
+        )
+
+        writer = WriteGear(output_filename=video_path, logging=True)
+        writer.execute_ffmpeg_cmd(self.get_command(video_path, output_path))
+        writer.close()
+
+    @abstractmethod
+    def get_command(
+        self, video_path: str, output_path: Optional[str] = None, **kwargs
```

Comment:
```suggestion
        self, video_path: str, output_path: Optional[str] = None
```
small nit: don't think we actually need kwargs here; if there are other args to use in the command let's set them as instance variables in `__init__` & then get them using `self.<var>` in `get_command()`.